presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -274,6 +274,13 @@ public void dropSchema(JdbcIdentity identity, String schemaName)
         invalidateSchemasCache();
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
```

Comment:
What `Optional.empty()` means? A `null`?

Reply:
Yes, it means `null`.